servlet/src/main/scala/org/http4s/servlet/Http4sServlet.scala

```diff
@@ -20,7 +20,7 @@ import org.log4s.getLogger
 class Http4sServlet(service: HttpService,
                     asyncTimeout: Duration = Duration.Inf,
                     threadPool: ExecutorService = Strategy.DefaultExecutorService,
-                    private[this] var servletIo: ServletIo = NonBlockingServletIo(4096))
+                    private[this] var servletIo: ServletIo = BlockingServletIo(4096))
```

Comment:
Another hard coded chunk size: maybe this should be a constant defined in an object? Its not a real issue, just noticing due to the discrepency in the ```ServletContextSyntax```.